htdocs/projet/class/task.class.php

```diff
@@ -479,13 +479,36 @@ function ($allTasksCompleted, $task) {
 				$newdir = $conf->projet->dir_output.'/'.dol_sanitizeFileName($project->ref).'/'.dol_sanitizeFileName($this->ref);
 				if (file_exists($olddir)) {
 					include_once DOL_DOCUMENT_ROOT.'/core/lib/files.lib.php';
-					$res = dol_move($olddir, $newdir);
+					$res = @rename($olddir, $newdir);
```

Comment:
Why replacing dol_move with rename ?

Reply:
Because the sourcefile of dol_move() cannot be a directory
@param string $srcfile Source file (can't be a directory. use native php @rename() to move a directory)

Reply:
Thanks to introduce a dol_move_dir() including renaming of ref.